testing/trino-product-tests/src/main/java/io/trino/tests/product/hudi/TestHudiSparkCompatibility.java

```diff
@@ -185,6 +185,41 @@ public void testCopyOnWriteTableSelectAfterUpdate()
         }
     }
 
+    @Test(groups = {HUDI, PROFILE_SPECIFIC_TESTS})
+    public void testCopyOnWriteTableContainsReplaceCommit()
+    {
+        String tableName = "test_hudi_cow_replace_commit_" + randomNameSuffix();
+
+        createNonPartitionedTable(tableName, COW_TABLE_TYPE);
+
+        try {
+            onHudi().executeQuery(format(
+                    """
+                            ALTER TABLE default.%s
+                            SET SERDEPROPERTIES (
+                                hoodie.parquet.small.file.limit=0,
+                                hoodie.clustering.inline=true,
+                                hoodie.clustering.inline.max.commits=2)""",
+                    tableName));
+            assertThat(onHudi().executeQuery("show TBLPROPERTIES default." + tableName))
```

Comment:
I don't think it's necessary to validate the properties here. We can assume that `ALTER TABLE` works correctly.

Reply:
Removed it